dds/idl/marshal_generator.cpp

```diff
@@ -1339,9 +1339,12 @@ namespace {
       extraction.addArg("strm", "Serializer&");
       extraction.addArg("arr", wrapper.wrapped_type_name());
       extraction.endArgs();
-      be_global->impl_ <<
-        "  bool discard_flag = false;\n"
-        "  const Encoding& encoding = strm.encoding();\n"
+	  
+	  if (!primitive) {
```

Comment:
We don't accept tabs or trailing whitespace on lines. [Here's a list of the whitespace issues](https://github.com/objectcomputing/OpenDDS/pull/2962/checks?check_run_id=3489990433).

Reply:
Tabs and trailing white space removed.